### PR TITLE
Add article PDF

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: vendor-extra/JatsContentBundle/src/EventListener/BuildView/FrontPdfContentHeaderListener.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        if (!$pdf instanceof Element) {
+            return $view;
+        }
+
+        $uri = UriResolver::resolve(
+            uri_for($pdf->baseURI),
+            uri_for($pdf->getAttributeNS('http://www.w3.org/1999/xlink', 'href'))
+        );
+
+        if (!in_array($uri->getScheme(), ['http', 'https'], true)) {
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/src/EventListener/BuildView/FrontPdfContentHeaderListener.php]: what else could this be? `ftp://`?

Comment (thewilkybarkid) [reply vendor-extra/JatsContentBundle/src/EventListener/BuildView/FrontPdfContentHeaderListener.php]: Yes, or potentially relative.

File: vendor-extra/JatsContentBundle/tests/EventListener/BuildView/FrontPdfContentHeaderListenerTest.php [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+                    'href' => 'http://example.com/file.pdf',
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * @test
+     */
+    public function it_does_nothing_if_the_uri_is_relative() : void
```
Comment (giorgiosironi) [inline vendor-extra/JatsContentBundle/tests/EventListener/BuildView/FrontPdfContentHeaderListenerTest.php]: is this meant to be replaced later? Or just not supported?

Comment (thewilkybarkid) [reply vendor-extra/JatsContentBundle/tests/EventListener/BuildView/FrontPdfContentHeaderListenerTest.php]: Not supported. No where to load it from.

**Pull request merged**